envoyauth/request.go

```diff
@@ -32,6 +33,7 @@ var v3Info = ast.NewObject(
 	keyValue("ext_authz", "v3"),
 	keyValue("encoding", "protojson"),
 )
+var ErrInvalidPath = errors.New("invalid parsed path")
```

Comment:
Do you think we need to export this? I think we can rename it `errInvalidPath` and avoid making it part of the package's interface.